### PR TITLE
835 svn role

### PR DESCRIPTION
**Pull request by kayiwa**

### CODE REVIEW

File: roles/pulibrary.svn/tasks/main.yml [hunk 1]
```diff
@@ -1,2 +1,6 @@
 ---
 # tasks file for roles/pulibrary.svn
+- name: add mod svn
+  apt:
+    name: ["libapache2-mod-svn"]
```
Comment (escowles) [inline roles/pulibrary.svn/tasks/main.yml]: Aren't we using `svnserve` instead of `mod_dav_svn`?

Comment (kayiwa) [reply roles/pulibrary.svn/tasks/main.yml]: indeed. was "cargo culting" this documentation.

Comment (escowles) [reply roles/pulibrary.svn/tasks/main.yml]: 👍

**Comment by kayiwa** [comment]
heya @escowles my hope (sorry I didn't explain that) is that this role would install svn and a separate one would administer it. This is a pattern @carolyncole and I feels allows us to scale. So one can always install subversion

**Comment by escowles** [comment]
@kayiwa Oh, that makes total sense!

**Pull request merged**